apps/guide/content/docs/legacy/app-creation/project-setup.mdx

```diff
@@ -26,72 +26,47 @@ const { token } = require('./config.json');
 console.log(token);
 ```
 
-<Callout title="Attention!" type="error">
-	If you're using Git, you should **not commit** this file and should ignore it via by using a `.gitignore` file as
-	explained below.
+<Callout title="Danger" type="error">
+	If you're using Git, you should not commit files containing secrets. Read on to find out how to [exclude them from
+	versioning by using `.gitignore`](#git-and-gitignore).
 </Callout>
 
 ## Using environment variables
 
-Environment variables are special values for your environment (e.g., terminal session, Docker container, or environment variable file). You can pass these values into your code's scope so that you can use them.
-
-One way to pass in environment variables is via the command line interface. When starting your app, instead of `node index.js`, use `TOKEN=your-token-goes-here node index.js`. You can repeat this pattern to expose other values as well.
-
-You can access the set values in your code via the `process.env` global variable, accessible in any file. Note that values passed this way will always be strings and that you might need to parse them to a number, if using them to do calculations.
-
-```sh tab="Command Line"
-A=123 B=456 DISCORD_TOKEN=your-token-goes-here node index.js
-```
-
-```js tab="Usage"
-console.log(process.env.A);
-console.log(process.env.B);
-console.log(process.env.DISCORD_TOKEN);
-```
-
-### Using dotenv
-
-Another common approach is storing these values in a `.env` file. This spares you from always copying your token into the command line. Each line in a `.env` file should hold a `KEY=value` pair.
-
-You can use the [`dotenv` package](https://www.npmjs.com/package/dotenv) for this. Once installed, require and use the package to load your `.env` file and attach the variables to `process.env`:
-
-```sh tab="npm"
-npm install dotenv
+Environment variables are, as the name suggets, values you can pass to your environment (e.g. terminal session, Docker container, node process). This has the benefit that you can keep your code the same for different execution contexts.
 
+```txt title=".env"
+A=Hello World
+B=123
+DISCORD_TOKEN=MTI3NDMxMjA3PDQ3ODIxNzIzNg.G6uEbl.IpA3-9YeScYr9pu9K1utMlpP4p-KJwNxcIAbi8
 ```
 
-```sh tab="yarn"
-yarn install dotenv
-```
+<Callout title="Danger" type="error">
+	If you're using Git, you should not commit `.env` or other environment files containing secrets. Read on to find out
+	how to [exclude them from versioning by using `.gitignore`](#git-and-gitignore).
+</Callout>
 
-```sh tab="pnpm"
-pnpm add dotent
-```
+To use environment variables in Node.js, we recommend you use the command line interface flag `--env-file` to point to the `.env` file you want to use. Note that the file name `.env` is just a convention. You could for example have a `development.env` and `production.env` file with different values depending on the Discord application you want to run your code.
```

Comment:
use `.env.production`, `.env.development` instead to match the filenames `bun` would load automatically as mentioned in next callout.